src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -29,15 +29,27 @@ class DebugCommand extends Command
     private $twig;
 
     /**
-     * {@inheritdoc}
+     * @param Environment $twig
      */
     public function __construct($name = 'debug:twig')
```

Comment:
doc block and arg must have the same name for sure,
then I suggest $twig because that's the most future oriented, even if the default value looks strange.
One way to make it better would be to have null as default, and internally turn null into this string, in the deprecated code path. Might be the best in fact. (Same for the other command of course).